src/main/scala/com/databricks/spark/xml/XmlOptions.scala

```diff
@@ -57,12 +57,15 @@ private[xml] class XmlOptions(
     logger.warn(s"$parseMode is not a valid parse mode. Using ${ParseModes.DEFAULT}.")
   }
 
+  if (attributePrefix.isEmpty) {
+    logger.warn("'attributePrefix' option should not be empty string.")
+  }
+
   val failFast = ParseModes.isFailFastMode(parseMode)
   val dropMalformed = ParseModes.isDropMalformedMode(parseMode)
   val permissive = ParseModes.isPermissiveMode(parseMode)
 
   require(rowTag.nonEmpty, "'rowTag' option should not be empty string.")
-  require(attributePrefix.nonEmpty, "'attributePrefix' option should not be empty string.")
```

Comment:
@BioQwer, BTW, mind if I ask why we warn here instead of the exception?

Reply:
@HyukjinKwon For my xml i need empty attributePrefix, because all my values in attributes.
My task it's use fias (All addresses of Russia) xml in spark.
I think many Russains will collide, in this problem.